Article/Article.js

```diff
@@ -1,20 +1,21 @@
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(currentArticle) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.currentArticle = currentArticle;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.currentArticle.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = "expand";
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
```

Comment:
I hope it makes sense that if you didn't use an arrow function here you would need to explicitly bind `this`
```
this.expandButton.addEventListener('click', this.expandArticle);
this.expandArticle = this.expandArticle.bind(this);
```